### PR TITLE
Add warning not to standardize a constant predictor.

### PR DESCRIPTION
**Pull request by arturgower**

### CODE REVIEW

File: src/Lasso.jl [hunk 1]
```diff
@@ -301,6 +301,10 @@ function StatsBase.fit{T<:AbstractFloat,V<:FPVector}(::Type{LassoPath},
     # Standardize predictors if requested
     if standardize
         Xnorm = vec(full(std(X, 1, corrected=false)))
+        if findfirst(Xnorm .== zero(T)) != 0
```
Comment (simonster) [inline src/Lasso.jl]: This could be more efficiently written as `any(x->x == 0, Xnorm)`

File: src/Lasso.jl [hunk 2]
```diff
@@ -301,6 +301,10 @@ function StatsBase.fit{T<:AbstractFloat,V<:FPVector}(::Type{LassoPath},
     # Standardize predictors if requested
     if standardize
         Xnorm = vec(full(std(X, 1, corrected=false)))
+        if findfirst(Xnorm .== zero(T)) != 0 
+            warn("One of the predicators (columns of X) is a constant, so it can not be standardized. 
+                  To include a constant predicator set standardize = false and intercept = false")
```
Comment (simonster) [inline src/Lasso.jl]: To avoid printing the second line with an indent, `"""` quotes might be better than `"`.

File: src/Lasso.jl (hunk 1 above)
Comment (arturgower) [inline src/Lasso.jl]: Thanks for the comments @simonster, they help me learn! I've implemented them.

**Comment by simonster** [comment]
Thanks!

**Pull request merged**